test/sanity/validate-modules/ignore.txt

```diff
@@ -774,10 +766,6 @@ lib/ansible/modules/remote_management/ucs/ucs_vnic_template.py E326
 lib/ansible/modules/remote_management/ucs/ucs_vsans.py E322
 lib/ansible/modules/remote_management/ucs/ucs_wwn_pool.py E322
 lib/ansible/modules/remote_management/ucs/ucs_wwn_pool.py E323
-lib/ansible/modules/source_control/github_deploy_key.py E336
```

Comment:
@dagwieers this PR doesn't change the `github_deploy_key` module; I think this line needs to remain in the ignore.txt file.

Reply:
Originally it did.

Reply:
The main problem here is that these PR's are being picked to shreds because one of the changes does make a code modification (for good reason) and I end up splitting them up. The amount of time I spend on things like these compared to the actual change is insane.

This was 2 months ago, when I still cared about having good looking module documentation.